ko/admin/config.rst

```diff
@@ -561,6 +565,57 @@ CUBRID 설치 시 생성되는 기본 데이터베이스 환경 설정 파일(**
 **tcp_keepalive** 
   
     **tcp_keepalive**\ 는 TCP 네트워크 프로토콜에 SO_KEEPALIVE 옵션을 적용할지 여부를 지정하는 파라미터로, 기본값은 **yes*\ 이다. 이 값이 **no**\ 이면 마스터 노드와 슬레이브 노드 간 방화벽이 설정되어 있는 환경에서 장시간 동안 트랜잭션 로그가 복사되지 않을 때 DB 서버 쪽 연결이 종료되는 현상이 발생할 수 있다. 
+
+**use_user_hosts** 
+
+    **use_user_hosts** 시스템 파라미터는 CUBRID 서비스에서 필요한 **호스트명**, **IP 주소** 검색 기능을 아래 서비스 중에서 선택하기 위해 사용한다. 기본값은 **OFF** 이다.
+
+    *   **시스템** 에서 제공하는 호스트, IP 주소 검색 라이브러리 (Linux의 경우 **glibc**)
+    *   **CUBRID** 에서 제공하는 **호스트/IP 주소 검색 라이브러리** 
+
+    use_user_hosts=off (기본값)
+
+    * 시스템 라이브러리를 사용하여 IP 주소, 호스트명을 검색한다.
+    * 일반적으로 /etc/hosts 파일 검색, DNS Query 등이 사용된다.
+
+    use_user_hosts=on
+
+    * CUBRID 호스트 검색 라이브러리를 이용하여 IP 주소/호스트명을 검색한다.
+    * **$CUBRID/conf/cubrid_hosts.conf** 파일을 이용하여,  호스트명과 IP 주소을 검색한다
+    * 시스템의 /etc/hosts, /etc/nsswitch.conf 파일에 대한 읽기 권한 등과 관계 없이 동작한다.
+
+    .. warning::
+
+        **use_user_hosts** 파라미터는 CUBRID 서비스가 종료된 상태에서 변경하는 것을 권고한다. 서비스가 운영중인 상태에서 파라미터가 변경되면 정상적으로 서비스가 종료되지 않을 수 있다.
+
+    *   **$CUBRID/conf/hosts.conf** 파일의 포맷는 **/etc/hosts** 와 같으나 다음과 같은 제약이 있다.
+
+        * **IPv4** 주소 만을 허용한다 (**IPv6** 는 허용하지 않는다).
+        * 아래와 같이 공식 호스트명 이외의  **alias** 는 허용하지 않는다 ::
```

Comment:
```suggestion
        * 아래와 같은 형식의 **alias** 는 허용하지 않는다 ::
```

Reply:
please add descriptions and examples of acceptable alias formats.
example)
172.31.0.1 host1
172.31.0.1 alias1

Reply:
```
* 아래와 같은 형식의 **alias** 는 허용한다 ::
172.31.0.1 host1
172.31.0.1 alias1
```

Is this description and example what we should need?
@mhoh3963

Reply:
I edited it